crates/store/re_data_loader/src/load_file.rs

```diff
@@ -23,6 +23,8 @@ pub fn load_from_path(
     // NOTE: This channel must be unbounded since we serialize all operations when running on wasm.
     tx: &Sender<LogMsg>,
 ) -> Result<(), DataLoaderError> {
+    use crate::DataLoaderSettings;
+
```

Comment:
Inline this, this looks weird just standing there on its own (and weird looking makes the reader thinks that weird things are happening).

Reply:
My LSP does weird things sometimes 🙃